staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -427,6 +427,15 @@ type Firmware struct {
 	// Settings to set the kernel for booting.
 	// +optional
 	KernelBoot *KernelBoot `json:"kernelBoot,omitempty"`
+	// Information that can be set in the ACPI table
+	ACPI *ACPI `json:"acpi,omitempty"`
+}
+
+type ACPI struct {
+	// SlicNameRef should match the volume name of a secret object. The data in the secret should
+	// be a binary blob that follows the ACPI SLIC standard, see:
+	// https://learn.microsoft.com/en-us/previous-versions/windows/hardware/design/dn653305(v=vs.85)
+	SlicNameRef string `json:"slicNameRef,omitempty"`
```

Comment:
Do we need to restrict this to x86? @alicefr @victortoso

Reply:
I don't think so. I'm double checking and I'll let you know.

Reply:
Hi @xpivarc, there is no explicit restriction. It should work form Windows on arm, albeit this was not tested.